bit_manipulation/Count_of_set_bits.cpp

```diff
@@ -0,0 +1,31 @@
+/** * @file
+ * @brief program to Count set bits in an integer 
+ * @details
+ * Time Complexity :- O(log n)
+ * Space complexity :- O(1)
+
+**/
+#include <iostream>
+using namespace std;

```

Comment:
using namespace std is considered bad practice.
Please make scope specific namespaces. [Reference](https://www.geeksforgeeks.org/using-namespace-std-considered-bad-practice/)